apps/web-giddh/src/app/shared/header/header.component.scss

```diff
@@ -153,7 +153,7 @@ a[vr-item]:focus {
     position: relative;
     font-family: "opensans-semibold";
 }
-.no-padding {
+.p-0 {
```

Comment:
Please check if this style is now required as we have global style for `padding: 0`